robottelo/cli/factory.py

```diff
@@ -1288,6 +1372,90 @@ def make_host(options=None):
 
 
 @cacheable
+def make_fake_host(options=None, isolated_mode=False):
+    """Wrapper function for make_host to pass all required options for creation
+    of a fake host
+
+    :param options: options to pass directly to make_host. May be equal to
+     `None` or contain some of entities to use (for example, `organization-id`
+     to create all sub-entities inside specific organization)
+    :param isolated_mode: set `True` to create all the required entities
+     like architecture, domain, location etc. Set `False` to use Satellite
+     predefined ones where it's possible. Defaults to `False`.
+    """
+    if options is None:
+        options = {}
+
+    if not isolated_mode:
+        # Check if entity name or id was passed. If not - use the default one
+        # with id = 1
+        for entity in (
```

Comment:
there are some default entities auto created after product install. like domain, arch etc but not all.